server/test/api/enterprises.js

```diff
@@ -0,0 +1,110 @@
+/*global describe, it, beforeEach, process*/
+
+var chai = require('chai');
+var chaiHttp = require('chai-http');
+var expect = chai.expect;
+chai.use(chaiHttp);
+
+// workaround for low node versions
+if (!global.Promise) {
+  var q = require('q');
+  chai.request.addPromises(q.Promise);
+}
+
+// environment variables - disable certificate errors
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = 0;
+
+// base URL
+var url = 'https://localhost:8080';
+var user = { username : 'superuser', password : 'superuser', project: 1};
+
+/**
+* The /Enterprises API endpoint
+*
+* This test suite implements full CRUD on the /Enterprises HTTP API endpoint.
+*/
+describe('The /Enterprises API endpoint', function () {
+  var agent = chai.request.agent(url);
```

Comment:
Instead of using this hard-coded URL, use the url found in the `helpers` module.  Like this:
```js
var agent = chai.request.agent(helpers.baseUrl);
```
now you can delete the `url` variable defined above.